TIS-100/20176.0.txt

```diff
@@ -0,0 +1,31 @@
+@0
+
+@1
+MOV UP, ACC  # reads a value from the input and saves it to the accumulator
+SUB RIGHT  # subtracts accumulator with the left
+MOV ACC, DOWN  #  moves accumulator down
+NEG  #  negates accumulator (makes it either positive or negative)
```

Comment:
Comments should explain "why" rather than "what". Explain why the accumulator is negated here